mindee/endpoints.py

```diff
@@ -140,6 +140,69 @@ def predict_req_post(
             )
         return response
 
+    def predict_async_req_post(
+        self,
+        input_source: Union[LocalInputSource, UrlInputSource],
+        include_words: bool = False,
+        close_file: bool = True,
+        cropper: bool = False,
+    ) -> requests.Response:
+        """
+        Make an asynchronous request to POST a document for prediction.
+
+        :param input_source: Input object
+        :param include_words: Include raw OCR words in the response
+        :param close_file: Whether to `close()` the file after parsing it.
+        :param cropper: Including Mindee cropping results.
+        :return: requests response
+        """
+        data = {}
+        if include_words:
+            data["include_mvision"] = "true"
+
+        params = {}
+        if cropper:
+            params["cropper"] = "true"
+
+        if isinstance(input_source, UrlInputSource):
+            data["document"] = input_source.url
+            response = requests.post(
+                f"{self._url_root}/predict_async",
+                headers=self.base_headers,
+                data=data,
+                params=params,
+                timeout=self._request_timeout,
+            )
+        else:
+            files = {"document": input_source.read_contents(close_file)}
+            response = requests.post(
+                f"{self._url_root}/predict_async",
+                files=files,
+                headers=self.base_headers,
+                data=data,
+                params=params,
+                timeout=self._request_timeout,
+            )
```

Comment:
Should be able to make a separate function for this, used by `predict_async` and `predict` routes.
Looks like just the URL would need to change...